sirepo/package_data/approved_role_email.jinja

```diff
@@ -0,0 +1,2 @@
+Your request for {{ role }} access has been approved.
```

Comment:
I would name this file to be something auth_role_moderation/approve_email.jinja. It's a mistake to have too many top level templates.

Reply:
84a51a4c4d292b403b922c73411c9bad58de76bf